app/controllers/comment_controller.php

```diff
@@ -0,0 +1,28 @@
+<?php 
+Class CommentController extends AppController
+{
+     public function most_liked()
+    {
+        $comments = Comment::get_most_liked();
+
+        $this->set(get_defined_vars());
+    }
+
+    public function delete()
+    {
+        $comment = Comment::get(Param::get('comment_id'));
```

Comment:
I think the correct process should be checked for this:
1. Check if the comment exists
2. Check if the comment author and currently logged in user is the same.
3. Delete the comment
4. Notify the user that the comment is successfully deleted.

You may do this po if you have time :bow: